### ISSUE TITLE
Save lat and lon bound arrays in Calc outputs

### ISSUE THREAD

**Issue by spencerahill** [spencerahill/aospy#215]
Similar-but-different to #201 and #203.  The bound arrays would be useful to have for subsequent finite volume operations.  Plus it's congruent with our larger overall mission to embed as much metadata as possible into everything.

I haven't checked yet to determine exactly where in the `Calc.compute` pipeline the bound arrays are being dropped.

**Comment by spencerkclark** [comment]
I agree this could be helpful; it's a little tricky given xarray's data model.  Because of the behavior described below, this has a little bit of the flavor of #94 (because it might require storing the initial metadata somewhere and restoring it later).

> I haven't checked yet to determine exactly where in the Calc.compute pipeline the bound arrays are being dropped.

This will happen wherever a single DataArray (that doesn't have `BOUNDS_STR` as a dimension) is selected from the initially read-in Dataset (so whenever `DataLoader.load_variable` is called).  See https://github.com/spencerahill/aospy/issues/14#issuecomment-150797922 for an explicit example.

**Comment by spencerahill** [comment]
>this has a little bit of the flavor of #94 (because it might require storing the initial metadata somewhere and restoring it later)

I agree.

>This will happen wherever a single DataArray (that doesn't have BOUNDS_STR as a dimension) is selected from the initially read-in Dataset (so whenever DataLoader.load_variable is called). See #14 (comment) for an explicit example.

That's right, thanks.  So yes, very much akin to #94, it seems that we'll need to store the bounds arrays upon loading the data from disk, retain them e.g. as a `Calc._bounds` attribute, and then append them to the output DataArrays prior to serializing.  So that seems relatively straightforward.
